ProjectTemplates/templates/NetCord.GenericHost/NetCord.Template.Bot.csproj

```
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+    <PropertyGroup>
+<!--#if (framework == "net9.0") -->
+        <TargetFramework>net9.0</TargetFramework>
+<!--#else -->
+        <TargetFramework>net8.0</TargetFramework>
+<!--#endif -->
+        <Nullable>enable</Nullable>
+        <ImplicitUsings>enable</ImplicitUsings>
+    </PropertyGroup>
+
+    <ItemGroup>
+<!--#if (framework == "net9.0") -->
+        <PackageReference Include="Microsoft.Extensions.Hosting" Version="9.0.5" />
```

Might be better to use wildcards for the package references so the template is always pulling in the latest version by default

```suggestion
        <PackageReference Include="Microsoft.Extensions.Hosting" Version="9.*" />
```